semgrep-core/src/experiments/synthesizing/Pattern_from_Diff.ml

```diff
@@ -0,0 +1,128 @@
+module In = Input_to_core_j
+module Out = Output_from_core_j
+module R = Range
+module E = Entity_code
+module G = Graph_code
+module Set = Set_
+
+(*****************************************************************************)
```

Comment:
all this code needs to move to deep-semgrep now that Graph_code_AST has been moved there.
There is a src/experiments/ there too!

Reply:
I'm going to merge this first but will do